packages/material-ui/src/styles/createSpacing.js

```diff
@@ -1,3 +1,5 @@
+import { getTransformer } from '@material-ui/system';
```

Comment:
I wonder about the best public name for this module.

Reply:
`createUnarySpacing`?

Reply:
Sounds like a good name.

Reply:
I will change it